packages/react-ui/src/theme/sections/palette.js

```diff
@@ -171,29 +171,28 @@ export const commonPalette = {
   },
 
   // Custom common colors
-  qualitative: {
-    ...baseColors.qualitative
-  },
   default: {
-    light: baseColors.neutral[100],
     main: baseColors.neutral[200],
     dark: baseColors.neutral[50],
-    // are we missing here relatedLight ??
+    light: baseColors.neutral[100],
     contrastText: baseColors.common.black,
-    background: baseColors.shades.dark[8]
+    outlinerBorder: baseColors.shades.dark[25],
+    filledBackground: baseColors.shades.dark[4]
+  },
+  cartoBrand: {
+    naviBlue: '#162945',
```

Comment:
typo?
```suggestion
    navyBlue: '#162945',
```

Reply:
or is it for navigation?

Reply:
Yep, typo. This review was just a draft, so I'd planned to do a final check before submitting it for real review, but good catch, thanks :)